src/network/networkd-link.c

```diff
@@ -1254,6 +1254,27 @@ static int link_set_bridge_fdb(Link *link) {
         return 0;
 }
 
+static int link_set_bridge_mdb(Link *link) {
+        MdbEntry *mdb_entry;
+        int r;
+
+        if (!link->network)
+                return 0;
+
+        if (!link->network->bridge) {
+                log_link_error(link, "Cannot configure MDB entries on non-bridge port");
```

Comment:
Please check `LIST_IS_EMPTY(link->network->static_mdb_entries)` before this. Otherwise, many error logs are produced. Also, please use `log_link_warning()` for non-critical errors.
```
if (LIST_IS_EMPTY(link->network->static_mdb_entries))
        return 0;

if (!link->network->bridge) {
        log_link_warning(link, "...");
        return 0;
}
```